components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,85 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/layout';
+
+@include exports("detail-page-sidebar") {
+  .bx--detail-page-sidebar__container {
+    margin: 0 5% 0 0;
+    .details-description,
+    .tag-description {
+      font-size: 0.875rem;
+      word-wrap: break-word;
+    }
+    .tag-description {
+      margin: 0 0 2rem 0;
+      > span {
+        font-weight: 600;
+      }
+    }
+    .description-label {
+      margin: 0;
+      overflow: visible;
+      color: $support-04;
+    }
+    .description-tag-docs-link {
+      display: block;
+    }
+  }
+
+  .bx--artifact-details-sidebar-container {
+    .description-label {
+      margin: 0 0 1rem 0;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+    .details-container {
+      margin: 2rem 0 2rem 0;
+      .inset-container {
+        margin: 0 0 .5rem 0;
+        line-height: 1.5;
+        font-size: .875rem;
+        .label {
```

Comment:
Could you see if there's a way to untangle the css here? You're nested I think five layers deep, if you swap to a BEM structure instead i.e.

`.inset-container`
`.inset-container__label`

You'll cut down the specificity drastically. Additionallly, all Carbon class names are prefixed with `bx--`